cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -588,6 +588,34 @@
     </description>
   </property>
 
+  <property>
+    <name>app.run.records.table.ttl.age.days</name>
+    <value>0</value>
+    <description>
+      Amount of time (in days) to retain run records for old job runs. Set to 0
+      to retain them indefinitely.
+    </description>
+  </property>
+
+  <property>
+    <name>app.run.records.table.ttl.frequency.hours</name>
+    <value>12</value>
+    <description>
+      How frequently (in hours) to check for old run records. Has no effect if
+      app.run.records.table.ttl.age.days is set to 0.
+    </description>
+  </property>
+
+  <property>
+    <name>app.run.records.table.ttl.initial.delay.minutes</name>
+    <value>10</value>
+    <description>
+      How long (in minutes) to wait after initial start to initially scan for
+      old run records to delete. Has no effect if
```

Comment:
similar comment here, does 0 mean it scans immediately on start?

Reply:
setting this value `app.run.records.table.ttl.initial.delay.minutes` to 0 would immediately start this, yes. 

Do you think this is worth clarifying in the description?